Thanks for this, it looks like the failure is due to a health check done by hypothesis:

```
  =================================== FAILURES ===================================
  _______________________ TestAsymmetricGame.test_equality _______________________
  
  self = <axelrod.tests.unit.test_game.TestAsymmetricGame testMethod=test_equality>
  
      @given(asymgame1=asymmetric_games(), asymgame2=asymmetric_games())
  >   @settings(max_examples=5)
  E   hypothesis.errors.FailedHealthCheck: Data generation is extremely slow: Only produced 2 valid examples in 1.04 seconds (0 invalid ones and 2 exceeded maximum size). Try decreasing size of the data you're generating (with e.g. max_size or max_leaves parameters).
  E                 count | fraction |    slowest draws (seconds)
  E     asymgame1 |    4  |     63%  |      --   0.096,  0.169,  0.178,  0.208
  E     asymgame2 |    4  |     37%  |      --   0.044,  0.057,  0.091,  0.199
  E   See https://hypothesis.readthedocs.io/en/latest/healthchecks.html for more information about this. If you want to disable just this health check, add HealthCheck.too_slow to the suppress_health_check settings for this test.
```

Could you try disabling the healthcheck (as it suggests in the traceback). If not we can take a look at that.

--- a/axelrod/tests/unit/test_game.py
+++ b/axelrod/tests/unit/test_game.py
@@ -1,7 +1,7 @@
 import unittest
 
 import numpy as np
-from hypothesis import given, settings
+from hypothesis import HealthCheck, given, settings
 from hypothesis.extra.numpy import array_shapes, arrays
 from hypothesis.strategies import integers
 
@@ -123,7 +123,7 @@ class TestAsymmetricGame(unittest.TestCase):
         self.assertEqual(expected_repr, str(asymgame))
 
     @given(asymgame1=asymmetric_games(), asymgame2=asymmetric_games())
-    @settings(max_examples=5)
+    @settings(max_examples=5, suppress_health_check=(HealthCheck.too_slow,))
     def test_equality(self, asymgame1, asymgame2):
         """Tests equality of AsymmetricGames based on their matrices."""
         self.assertFalse(asymgame1 == "foo")

